versioneer.py

```diff
@@ -8,7 +8,7 @@
 * https://github.com/warner/python-versioneer
 * Brian Warner
 * License: Public Domain
-* Compatible With: python2.6, 2.7, 3.2, 3.3, 3.4, and pypy
+* Compatible With: python3.5+, and pypy
```

Comment:
Actually I suppose this should read Python 3.6+, but given this is part of the licensing (?) probably best to just leave it alone

Reply:
This is just documentation, not licensing, and we've made enough changes to this files as is.

Now that I look at this, I actually will just drop this line because I think we can safely assume that the file is going to be compatible with whatever versions of Python `pandas` itself supports.